bootstrap-salt.sh

```diff
@@ -3771,6 +3772,13 @@ install_debian_git_post() {
     done
 }
 
+install_debian_2021_post() {
```

Comment:
Should we name the function `install_kali_2021_post` I guess it would be more appropriate

Reply:
I don't know if it will work, because functions are loaded based on the OS name (which is debian here because kali is a debian flavour).